components/pkg-export-kubernetes/src/cli.rs

```diff
@@ -162,3 +162,18 @@ fn valid_natural_number(val: String) -> result::Result<(), String> {
         Err(_) => Err(format!("{} is not a natural number", val)),
     }
 }
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_valid_natural_number() {
+        valid_natural_number("99".to_owned()).unwrap();
+
+        for &s in ["x", "", "#####"].iter() {
```

Comment:
For my understanding, what's the point of `&s` here, instead of `s`?

Reply:
So not to take ownership and borrow as string slice. IIRC it didn't build without the "&".